src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -466,7 +466,7 @@ class CalculatorForm extends React.Component {
       <div>
         <RadioButtons
           label={this.renderLearnMoreLabel({
-            text: 'Will the majority of your classes be on campus?',
+            text: 'Will the majority of your classes be on main campus?',
```

Comment:
This is the request change @briangryth, should we add a "the" before main?

Reply:
Yes that would be better